devicetypes/smartthings/zwave-basic-smoke-alarm.src/zwave-basic-smoke-alarm.groovy

```diff
@@ -195,3 +223,35 @@ def zwaveEvent(physicalgraph.zwave.Command cmd, results) {
 	event.descriptionText = "$event.linkText: $cmd"
 	results << createEvent(event)
 }
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd, results)
+{
+	if (cmd.notificationType == 0x01) {  // Smoke Alarm
+		switch (cmd.event) {
+			case 0x00:
+			case 0xFE:
+				createSmokeEvents("smokeClear", results)
```

Comment:
Based on the notification command class spec, 0xFE means "unknown command". 0x00 means "previous events cleared". So this should be adjusted so that type 1 event 0 creates both a "smokeClear" and a "testedClear" event.

I don't think 0xFE should do anything at all.

Reply:
I realize now that "smokeClear" and "testClear" are functionally equivalent. So ignore me.